src/ast.rs

```diff
@@ -346,7 +392,43 @@ pub enum BinaryOpKind {
     BinaryAnd,
     BinaryOr,
 }
-
+impl Compile for BinaryOpKind {
+     fn compile(&self, t: &Target) -> String {
+        match self {
```

Comment:
Remove return keyword from all of these arms

Reply:
Additionally, change the return type to `&'static str` and remove all the `String::from` calls (Strings are heap allocated)